src/api/posts/.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  extends: '@senecacdot/eslint-config-telescope',
```

Comment:
We can extends more than one config.
```suggestion
  extends: ['@senecacdot/eslint-config-telescope'],
```